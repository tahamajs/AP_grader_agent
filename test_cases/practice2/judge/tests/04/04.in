reserve_table 26 2
reserve_table 28 4
reserve_table 3 3
reserve_table 27 3
reserve_table 1 3
reserve_table 29 5
reserve_table 30 6
reserve_table 2 1
reserve_table 9 1
reserve_table 32 2
reserve_table 5 2
reserve_table 21 2
reserve_table 47 3
reserve_table 19 2
reserve_table 18 3
reserve_table 7 2
reserve_table 15 3
reserve_table 16 2
reserve_table 10 4
reserve_table 17 2
reserve_table 50 1
reserve_table 41 5
reserve_table 45 6
reserve_table 49 2
reserve_table 14 1
reserve_table 38 4
reserve_table 11 3
reserve_table 25 5
reserve_table 23 4
reserve_table 36 1
