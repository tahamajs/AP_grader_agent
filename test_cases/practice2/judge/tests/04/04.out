Walter sits at table 2
Hank sits at table 4
Parsa sits at table 3
Jesse sits at table 3
Paria enters the waiting queue of table 3
Gustavo sits at table 5
Mike sits at table 6
Amir Hossein sits at table 1
Sadegh sits at table 1
Hasti sits at table 2
Fereshte sits at table 2
Tommy sits at table 2
Peyman enters the waiting queue of table 3
Sina enters the waiting queue of table 2
Amin enters the waiting queue of table 3
Mahdi enters the waiting queue of table 2
Erfan enters the waiting queue of table 3
Alex enters the waiting queue of table 2
Hamid sits at table 4
Sipamo enters the waiting queue of table 2
Sobhan sits at table 1
Melika sits at table 5
Marry sits at table 6
Parham enters the waiting queue of table 2
Babak sits at table 1
Aryan enters the waiting queue of table 4
Shaghayegh enters the waiting queue of table 3
Saman sits at table 5
Narges enters the waiting queue of table 4
Saeed enters the waiting queue of table 1
